com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 - Fixed NetworkObjects not being despawned before they are destroyed during shutdown for client, host, and server instances. (#1390)
 - Fixed client player object being destroyed on server when the client's player object has DontDestroyWithOwner set. (#1433)
+- Fixed: NetworkVariables containing more than 1300 bytes of data (such as large NetworkLists) no longer cause an OverflowException (the limit on data size is now whatever limit the chosen transport imposes on fragmented NetworkDelivery mechanisms) (#1481)
```

Comment:
_sigh..._ we also need to do better at handling transport errors within the SDK but that's for another day.